nautobot/docs/docker/index.md

```diff
@@ -0,0 +1,159 @@
+
+## Tags
+
+The Docker image is published to both the GitHub container registry as well as Docker Hub.
+
+To get the image from Docker Hub run:
+
+```no-highlight
+docker image pull networktocode/nautobot
+```
+
+!!! info
+    Effective November 20, 2020, [rate limits](https://www.docker.com/increase-rate-limits) have gone into effect on Docker Hub.  To increase the rate limit you can use `docker login` to authenticate with an account, however using the GitHub container registry avoids these limits all together.
+
+To get the image from the GitHub Container Registry (GHCR) run:
+
+```no-highlight
+$ docker image pull ghcr.io/nautobot/nautobot
+```
+
+The following tags are available:
+
+* `X.Y.Z` these images are built with the same baseline as the released Python packages based on the default python version (3.6) docker container
+* `latest` these images are built from the latest code in the main branch (should be the latest released version) based on the default python version (3.6) docker container
+* `X.Y.Z-py${PYTHON_VER}` these images are built with the same baseline as the released Python packages based on the python version ($PYTHON_VER) docker container
+* `latest-py${PYTHON_VER}` these images are built from the latest code in the main branch (should be the latest released version) based on the python version ($PYTHON_VER) docker container
+* `develop` these images are built from the latest code in the develop branch on each commit based on the default python version (3.6) docker container
+* `develop-${GIT_SHA:0:7}-$(date +%s)` tags for each commit to the develop branch based on the default python version (3.6) docker container
+* `develop-latest-py${PYTHON_VER}` these images are built from the latest code in the develop branch on each commit based on the python version ($PYTHON_VER) docker container
+* `develop-${GIT_SHA:0:7}-$(date +%s)-py${PYTHON_VER}` tags for each commit to the develop branch based on the python version ($PYTHON_VER) docker container
+
+To pull a specific tag you can append the image name with `:tag` for example, to pull the 1.0.0 image:
+
+```no-highlight
+$ docker image pull networktocode/nautobot:1.0.0
+```
+
+or to pull the 1.0.0 image from GitHub:
+
+```no-highlight
+$ docker image pull ghcr.io/nautobot/nautobot:1.0.0
+```
+
+Currently images are pushed for the following python versions:
+
+* 3.6
+* 3.7
+* 3.8
+* 3.9 _For Testing Only_
+
+!!! info
+    A dev image `networktocode/nautobot-dev` and `ghcr.io/nautobot/nautobot-dev` is also provided with the same tags, this image provides the development dependencies needed to build Nautobot.  This container can be used as a base for development to develop additional Nautobot plugins but should **NOT** be used in production.
+
+## Getting Started
+
+Nautobot requires a PostgreSQL database and Redis instance before it will start, because of this the quickest and easiest way to get Nautobot running is with `docker-compose`.
```

Comment:
```suggestion
Nautobot requires a PostgreSQL database and Redis instance before it will start. Because of this the quickest and easiest way to get Nautobot running is with `docker-compose`.
```